partials/contacts/new.html

```diff
@@ -1,63 +1,53 @@
-<div class="text-center" id="new">
+<div class="text-center" id="new" ng-controller="NewCtrl">
```

Comment:
You don't need to declare your `ng-controller` here since the pairing of your new.html partial and your NavCtrl occurs in your routing